docs/docs-components/Header.js

```diff
@@ -240,7 +245,7 @@ function Header() {
         </Box>
 
         <DocSearch
-          anchorRef={anchorRef}
+          anchorRef={searchAnchorRef}
```

Comment:
This was the cause of the jumping Dropdown, they were sharing one anchor ref

Reply:
Good find!